cli/flags/util.go

```diff
@@ -9,3 +13,28 @@ func eachName(longName string, fn func(string)) {
 		fn(name)
 	}
 }
+
+// MarkRequired marks flags with specified names as required.
+func MarkRequired(flagSet []cli.Flag, names ...string) []cli.Flag {
```

Comment:
Wouldn't it be easier to have a second variable defined where needed? This PR was supposed to reduce the number of code lines, not increase it.